examples/flutter_gallery/pubspec.yaml

```diff
@@ -3,13 +3,13 @@ dependencies:
   flutter:
     sdk: flutter
   collection: 1.14.6
-  device_info: 0.1.2
+  device_info: 0.2.0
   intl: 0.15.2
-  connectivity: 0.2.1
+  connectivity: 0.3.0
```

Comment:
Somehow after this change I'm getting error when running flutter_gallery:
```
╰─➤  cd $FH/flutter/examples/flutter_gallery 
╰─➤  ../../bin/flutter packages get
Running "flutter packages get" in flutter_gallery...             
Package connectivity has no versions that match 0.3.0 derived from:
- flutter_gallery depends on version 0.3.0
pub get failed (1)
```
Is this a known issue with know solution?

Reply:
In this PR have dependencies been updated manually rather than by using `flutter update-packages --force-upgrade`?
I see we still depend on `connectivity: 0.2.1` in dev/benchmarks/mega_gallery/pubspec.yaml.

Reply:
No, I used `--force-upgrade`.

Reply:
I've tried this on three machines - on all three when I run `flutter update-packages --force-upgrade` at the tip of the tree it reverts all three package deps:
```
diff --git a/examples/flutter_gallery/pubspec.yaml b/examples/flutter_gallery/pubspec.yaml
index 32d75c939..1609eebfb 100644
--- a/examples/flutter_gallery/pubspec.yaml
+++ b/examples/flutter_gallery/pubspec.yaml
@@ -3,13 +3,13 @@ dependencies:
   flutter:
     sdk: flutter
   collection: 1.14.6
-  device_info: 0.2.0
+  device_info: 0.1.2
   intl: 0.15.2
-  connectivity: 0.3.0
+  connectivity: 0.2.1
   string_scanner: 1.0.2
-  url_launcher: 3.0.0
+  url_launcher: 2.0.2
   cupertino_icons: 0.1.1
-  video_player: 0.3.0
+  video_player: 0.2.1
 
   # Also update dev/benchmarks/complex_layout/pubspec.yaml
   flutter_gallery_assets:
```

Reply:
well, maybe something changed on `pub`. Will you submit PR to undo it?

Reply:
I would do it, but if nobody else seem to have problem with it(I mean I can't run flutter gallery example at ToT unless I do flutter update-package --force-upgrade to revert this change), I'm hesitant.
Does flutter gallery at ToT works for you?

Reply:
Just checked with @a-siva and 0.3.0 works for him, `flutter update-packages --force-upgrade` doesn't revert changes, flutter-gallery example works as expected. I'm puzzled.

Reply:
@aam try purging your `~/.pub-cache`.

Reply:
yeah, @mraleph, tried that.

Reply:
For those who have this problem - make sure you get all the tags from flutter repo. In my case I was synching down only master branch(by doing `git pull upstream master`). What I had to do to fix the problem was to do `git pull upstream`.

Reply:
Fuchsia is also broken on 
```
Running "flutter packages get" in flutter_gallery...             
Package connectivity has no versions that match 0.3.0 derived from:
```